### PR TITLE
Configure pagopa api-config

### PR DESCRIPTION
**Pull request by pasqualedevita**

### CODE REVIEW

**Comment by pasqualedevita** [comment]
`healthcheck_container` is no more needed and deprecated. Now in new pipeline we use curl on azure self hosted agent. More info at https://github.com/pagopa/io-functions-app/pull/212

File: azure-devops/projects/pagopa-projects/pagopa-api-config.tf [hunk 1]
```diff
@@ -98,5 +116,32 @@ module "pagopa-api-config_deploy" {
...
   service_connection_ids_authorization = [
     azuredevops_serviceendpoint_github.io-azure-devops-github-ro.id,
+    azuredevops_serviceendpoint_azurerm.DEV-PAGOPA.id,
+    azuredevops_serviceendpoint_azurerm.UAT-PAGOPA.id,
+  ]
+}
+
+
+module "pagopa-api-config_newman-test" {
+  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_code_review?ref=v0.0.3"
```
Comment (pasqualespica) [inline azure-devops/projects/pagopa-projects/pagopa-api-config.tf:126]: if you use [this template](https://github.com/pagopa/azuredevops-tf-modules/blob/e991d898ce9130cfc2cf2dee999d2ec8d28fec7d/azuredevops_build_definition_code_review/main.tf#L26) you can't define your project pipeline `devops/newman-test-pipeline.yml` see [PR](https://github.com/pagopa/pagopa-api-config/pull/15).
I think could be better ( at the moment ) add a stage in the existing code-review pipeline

**Review by pasqualespica — CHANGES_REQUESTED** [review]

**Comment by pasqualespica** [comment]
@jacopocarlini if we does't have task jira, we can create new one. In this way we have all actions tracked

File: azure-devops/projects/pagopa-projects/pagopa-api-config.tf (hunk 1 above)
Comment (pasqualedevita) [reply azure-devops/projects/pagopa-projects/pagopa-api-config.tf:126]: absolutely, you can add a parallel stage to code review pipeline to be faster.
Automated test are very useful on code review, in this way you can fix possible errors before merge your PR (early feedback :) )

Comment (jacopocarlini) [reply azure-devops/projects/pagopa-projects/pagopa-api-config.tf:126]: Ok. I proceed in this way

**Comment by jacopocarlini** [comment]
> @jacopocarlini if we does't have task jira, we can create new one. In this way we have all actions tracked

The jira task exists. It is on our board

**Pull request merged**